installers/Launcher.hs

```diff
@@ -39,8 +39,12 @@ launcherArgs launcher = unwords $
         "--kademlia-peers-file", "ip-dht-mappings",
         "--system-start", "1499360281",
         "--wallet",
+        "--tlscert", quote (runtimePath launcher <> tlsPath <> "server" <> pathSeparator <> "server.crt"),
```

Comment:
@akegalj, seems good -- the certs are `tls/server/server.crt` and `tls/client/client.crt` -- relative to Daedalus executable.